contentcuration/contentcuration/utils/celery/tasks.py

```diff
@@ -311,31 +304,3 @@ def revoke(self, exclude_task_ids=None, **kwargs):
         # be sure the database backend has these marked appropriately
         self.TaskModel.objects.filter(task_id__in=task_ids).update(status=states.REVOKED)
         return count
-
-
-class CeleryAsyncResult(AsyncResult):
-    """
-    Custom result class which has access to task data stored in the backend
-
-    The properties access additional properties in the same manner as super properties,
-    and our custom properties are added to the meta via TaskResultCustom.as_dict()
-    """
-
-    def get_model(self):
-        """
-        :return: The TaskResult model object
-        :rtype: contentcuration.models.TaskResult
-        """
-        return get_task_model(self, self.task_id)
-
-    @property
-    def user_id(self):
-        return self._get_task_meta().get('user_id')
-
-    @property
-    def channel_id(self):
-        return self._get_task_meta().get('channel_id')
-
-    @property
-    def progress(self):
-        return self._get_task_meta().get('progress')
```

Comment:
This class should still exist, but the methods should proxy to the new model instead.